integrations/microsoft-teams/microsoft-teams-alerts/sensu-resources.yaml

```diff
@@ -0,0 +1,132 @@
+# Description
+#   The Sensu Microsoft Teams Handler is a Sensu Event Handler that alerts to
+#   the Microsoft Teams channels.
+#
+# Instructions
+#   1. Add "teams" to the "alert" handler set.
+#
+#      To create the "alert" handler set, please run:
+#
+#      $ sensuctl handler create alert --type set --handlers teams
+#
+#      If you already have a handler set called "alert", please run:
+#
+#      $ sensuctl edit handler alert
+#
+#      NOTE: the sensuctl edit command will open your default editor; please
+#      modify the "handlers" attribute by adding "teams".
+#
+#   2. Configure the following secrets using your preferred Sensu Go Secrets
+#      provider:
+#
+#      - microsoft_teams_webhook_url
+#
+#      NOTE: example secret resources using the "env" provider are included in
+#      this template; to add your secrets to the Sensu Backend, please modify
+#      /etc/default/sensu-backend or /etc/sysconfig/sensu-backend and restart
+#      the sensu-backend service.
+#
+#   3. More configuration options are available for this plugin; please see the
+#      plugin documentation for more details.
+#
+# Documentation
+#   - https://bonsai.sensu.io/assets/sensu/sensu-microsoft-teams-handler
+#   - https://docs.sensu.io/sensu-go/latest/reference/secrets/
+#
+# Contributors
+#   The following individuals have contributed to this configuration template:
+#   - Todd Campbell, @nixwiz
+---
```

Comment:
```suggestion
---
```

Reply:
No need to leave these comments here; also I'm pretty sure this is invalid YAML; comments need to live inside of a non-empty YAML document (i.e. there needs to be at least one field). 

I'd suggest just deleting the comments.